### PR TITLE
oslogin: port google_oslogin_control to FreeBSD

### PR DESCRIPTION
**Pull request by lucaskanashiro**

### CODE REVIEW

**Comment by lucaskanashiro** [comment]
Updated with the latest commits in development branch. @illfelder the CLA issue was not fixed yet, could you check that please?

**Comment by lucaskanashiro** [comment]
I signed it!

File: google_compute_engine_oslogin/bin/google_oslogin_control [hunk 1]
```diff
@@ -65,24 +88,29 @@ overwrite_file() {
...
+  $sed_bin -i "\$a${added_comment}\n${sshd_user}" ${sshd_config}.new
 }
 
 add_to_nss_config() {
   remove_from_nss_config
-  sed -i '/^passwd:/ s/$/ cache_oslogin oslogin/' ${nss_config}.new
+  $sed_bin -i '/^passwd:/ s/$/ cache_oslogin oslogin/' ${nss_config}.new
+
+  if is_freebsd; then
+    # Replace compat by files (as compat cannot be used with other sources)
```
Comment (illfelder) [inline google_compute_engine_oslogin/bin/google_oslogin_control]: Please start comments with capital letters and end comments with periods.

Comment (lucaskanashiro) [reply google_compute_engine_oslogin/bin/google_oslogin_control]: Fixed.

**Pull request merged**